overlord/snapstate/snapmgr.go

```diff
@@ -90,9 +90,10 @@ type SnapManager struct {
 // SnapSetup holds the necessary snap details to perform most snap manager tasks.
 type SnapSetup struct {
 	// FIXME: rename to RequestedChannel to convey the meaning better
-	Channel string `json:"channel,omitempty"`
-	UserID  int    `json:"user-id,omitempty"`
-	Base    string `json:"base,omitempty"`
+	Channel string   `json:"channel,omitempty"`
+	UserID  int      `json:"user-id,omitempty"`
+	Base    string   `json:"base,omitempty"`
+	Prereq  []string `json:"prereq,omitempty"`
```

Comment:
Can you document what the content of `Prereq` is? Are those snap names / snap IDs?